libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/attributeFilter/validation/uniqueFiltersValidation.ts

```diff
@@ -27,5 +27,9 @@ export async function canFilterBeAdded(
         loadExistingDisplayForms,
     ]);
 
-    return !existingDisplayForms.some((existing) => areObjRefsEqual(existing, addedDisplayForm));
+    const attributeFilterRef = addedDisplayForm.attribute;
+    const existingAttributes = existingDisplayForms.map((df) => df.attribute);
+
+    // lookup has to be throw related Attributes since we support only one DF for one Attribute
```

Comment:
This comment is confusing, it talks about a throw but it does not throw anything.

Reply:
fixed